ql/src/test/results/clientpositive/llap/dynamic_partition_pruning.q.out

```diff
@@ -4277,37 +4277,21 @@ STAGE PLANS:
                   alias: srcpart
                   filterExpr: ds is not null (type: boolean)
                   Statistics: Num rows: 2000 Data size: 389248 Basic stats: COMPLETE Column stats: COMPLETE
-                  Filter Operator
-                    predicate: ds is not null (type: boolean)
```

Comment:
Note that the filter operator is removed. We need to be careful here because not all input formats guarantee that the filter expression is being applied / does not return false positives. I would expect the Filter remains but only a single time?